### PR TITLE
Replace settings strings with constants

### PR DESCRIPTION
**Pull request by rak-phillip**
### Summary

This fixes a few instances where settings are referenced without using the constants. These were discovered while reviewing and validating changes for rancher/rancher#45894

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Review by mantis-toboggan-md — DISMISSED** [review]

File: shell/utils/version.js [hunk 1]
```diff
@@ -82,7 +83,7 @@ export function isDevBuild(version) {
 }
 
 export function getVersionInfo(store) {
-  const setting = store.getters['management/byId'](MANAGEMENT.SETTING, 'server-version');
+  const setting = store.getters['management/byId'](MANAGEMENT.SETTING, SETTING.SERVER_VERSION);
```
Comment (nwmac) [inline shell/utils/version.js:86]: e2e tests are failing - I don't think there is a `SERVER_VERSION`enum - I think you want `VERSION_RANCHER`

**Review by nwmac — CHANGES_REQUESTED** [review]

**Comment by nwmac** [comment]
@rak-phillip One comment, which I think is why the tests are failiong

File: shell/utils/version.js (hunk 1 above)
Comment (rak-phillip) [reply shell/utils/version.js]: Yep, definitely the wrong value used here. Thanks for the tip 🙂

**Comment by rak-phillip** [comment]
@nwmac tests all pass, ready for another review

**Review by nwmac — APPROVED** [review]
LGTM

**Pull request merged**